packages/transform-to-vanilla/src/transform-keys/simple-pseudo-selectors.ts

```diff
@@ -0,0 +1,57 @@
+// == Type ================================================================
+type DefaultType = string & NonNullable<unknown>;
+type PseudoSelectorsSign = `_${string}` | `__${string}`;
+type ReplacedPseudoSelectorsSign = `:${string}` | `::${string}`;
```

Comment:
It would be a good idea to refer to the following file.
- https://github.com/vanilla-extract-css/vanilla-extract/blob/master/packages/css/src/simplePseudos.ts

Reply:
woahoo every pseudos are organized, it's really good, thx👍